packages/core/stories/button.doc.stories.mdx

```diff
@@ -89,21 +95,29 @@ export const Link = ({ children, to, ...props }) => (
   </ComponentAnatomy>
 </Canvas>
 
-## CSS Class
+### CSS Class
 
 <CSSClassTable of={Button} />
 
-## Characteristics Used
+### Characteristics used
 
 <CharacteristicUsage of={Button} />
 
-## --uitkButton CSS Custom Property API
+### --uitkButton CSS custom property API
 
 The default styling values for most Button attributes are provided by the theme variables. A smaller number of attributes
 use hard-coded values local to Button. Both of these can be overridden, see Theme documentation for detailed usage guidance.
 
 The CSS custom properties below are consumed by Button, but not defined by Button. They can be defined via a container or
 a custom class name to override button styling. They will always take precedence over default styles, whether from theme variable
-or declared locally. Again, see Theme documentation for guidance on when to use variables from the Custom Property API.
+or declared locally. Again, see Theme documentation for guidance on when to use variables from the custom property API.
 
 <CSSVariableTable of={Button} />
+
+# Help and support
```

Comment:
We should make this into a block so it's in one place soon

Reply:
That would be great, how can we do it?

Reply:
It would be on our end. If I have some time I'll give it a go